frontend/src/components/common/header/styles.module.scss

```
@@ -0,0 +1,26 @@
+.header-wrapper {
+  display: flex;
+  flex-grow: 1;
+  justify-content: space-between;
+  height: 96px;
```

```suggestion
  min-height: 96px;
```
try to avoid fixed width/height values

---

got it